packages/labs/virtualizer/README.md

```diff
@@ -285,6 +285,65 @@ export class MyItems extends LitElement {
 }
 ```
 
+## ResizeObserver dependency
+
+Virtualizer depends on the standard [`ResizeObserver`]() API, which is supported in all modern browsers. In case your browser support matrix includes older browsers that don't implement `ResizeObserver`, the Virtualizer package includes a `ResizeObserver` polyfill that is known to be compatible with Virtualizer.
+
+### Using the default loader
+
+The package also includes a simple mechanism for loading the `ResizeObserver` polyfill. This mechanism uses feature detection to determine whether the polyfill is required; if it is, then the polyfill is automatically loaded and provided directly to Virtualizer.
+
+You need to invoke the loader—which is aysnchronous—and await its return before running any code that would cause a virtualizer to be instantiated. The simplest way to do this is to load virtualizer itself with a dynamic `import()` statement. For example:
+
+```js
+import {loadPolyfillIfNeeded} from '@lit-labs/virtualizer/polyfillLoaders/ResizeObserver.js';
+
+async function load() {
+  await loadPolyfillIfNeeded();
+  await import('@lit-labs/virtualizer');
```

Comment:
It might be instructive to show this happening at the app level, like before dynamic importing the main app code in a bootstrap script. You could use top-level await there.

Reply:
I considered that briefly, but was struggling with doing so in a way that didn't seem overly precise / prescriptive or make too many assumptions about typical app structure, so I ended up falling back to this minimal, more conceptual sketch. Feel free to propose (or just go ahead and make) a better example.